example/functions/create-session.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * This function creates a Stripe Checkout session and returns the session ID
+ * for use with Stripe.js (specifically the redirectToCheckout method).
+ *
+ * @see https://stripe.com/docs/payments/checkout/one-time
+ */
+
+const stripe = require('stripe')(process.env.REACT_APP_STRIPE_API_SECRET);
+
+/*
+ * Product data can be loaded from anywhere. In this case, we’re loading it from
+ * a local JSON file, but this could also come from an async call to your
+ * inventory management service, a database query, or some other API call.
+ *
+ * The important thing is that the product info is loaded from somewhere trusted
+ * so you know the pricing information is accurate.
+ */
+const inventory = require('./data/products.json');
+// const { validateCartItems } = useStripeCart;
+
+const validateCartItems = (inventorySrc, cartItems) => {
+  const validatedItems = Object.keys(cartItems).map((item) => {
+    const product = cartItems[item];
+    const validatedItem = inventorySrc.find(
+      (p) => p.name === cartItems[item].name
+    );
+    return {
+      name: validatedItem.name,
+      amount: validatedItem.price * product.quantity,
+      currency: validatedItem.currency,
+      quantity: product.quantity,
+    };
+  });
+
+  return validatedItems;
+};
+
+exports.handler = async (event) => {
+  const productJSON = JSON.parse(event.body);
+
+  const line_items = validateCartItems(inventory, productJSON);
+
+  const session = await stripe.checkout.sessions.create({
+    payment_method_types: ['card'],
+    billing_address_collection: 'auto',
+    shipping_address_collection: {
+      allowed_countries: ['US', 'CA'],
+    },
+
+    /*
+     * This env var is set by Netlify and inserts the live site URL. If you want
+     * to use a different URL, you can hard-code it here or check out the
+     * other environment variables Netlify exposes:
+     * https://docs.netlify.com/configure-builds/environment-variables/
+     */
+    success_url: `${process.env.URL}/success.html`,
```

Comment:
I'm assuming this is a page we need to provide - I notice we don't have a success.html yet.

Reply:
I don't think we should provide this, we should allow them to pass a string to their successful purchase page. Just gotta work it in

Reply:
@doingandlearning actually the success url stuff is not part of our library, it's teh code a deve would have to write on the server side. Although, it might be a nice to have if we can make this part easier, too. Not sure how that would look, though.

What do you think? 

Might not be the best approach, but I'm thinking that we can have a create session function where you pass the authenticated stripe object and use `stripe.checkout.sessions.create`. Kinda feels redundant though

Reply:
Fair enough - I guess I was just thinking if someone was kicking the tyres this is an incomplete example. Agreed though that this is not key functionality :)